src/components/Input/Input.tsx

```diff
@@ -16,10 +16,10 @@ interface InputProps {
   onChange?: (val: any) => void;
 
   // these come from react-aria
-  labelProps: LabelHTMLAttributes<HTMLLabelElement>;
-  inputProps: InputHTMLAttributes<HTMLInputElement>;
-  descriptionProps: HTMLAttributes<HTMLElement>;
-  errorMessageProps: HTMLAttributes<HTMLElement>;
+  labelProps?: LabelHTMLAttributes<HTMLLabelElement>;
+  inputProps?: InputHTMLAttributes<HTMLInputElement>;
+  descriptionProps?: HTMLAttributes<HTMLDivElement>;
+  errorMessageProps?: HTMLAttributes<HTMLDivElement>;
```

Comment:
Can we put this in a separate PR?

Reply:
I had to update it here because it causes the ERROR in zOS app